sklearn/preprocessing/imputation.py

```diff
@@ -8,13 +8,14 @@
 from scipy import sparse
 from scipy import stats
 
+from ..neighbors import KDTree, NearestNeighbors
```

Comment:
Could it be that this creates circular imports? I feel something odd is happening here. Do tests run on your box without any problems?